src/app/components/core/configuration-manager/configuration-manager.component.ts

```diff
@@ -15,9 +15,7 @@ import {
 
 export class ConfigurationManagerComponent implements OnInit {
   public categoryData = [];
-  public rootCategories = [];
   public JSON;
```

Comment:
What is this variable for?

Reply:
It seems, it doesn't have any use here but need to verify once more. It is as it was before.